posts/2023-03-07-23.0.0.2.adoc

```diff
@@ -0,0 +1,330 @@
+
+
+The following sample `server.xml` file enables the features to configure the Admin Center test connection function and configures the connection to the Derby database:
+
+[source, xml]
+----
+<server description="new server">
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>adminCenter-1.0</feature>
+        <feature>restConnector-2.0</feature>
+        <feature>jdbc-4.3</feature>
+        <feature>mpOpenApi-3.0</feature>
+    </featureManager>
+
+    <!-- To access this server from a remote client add a host attribute to the following element, e.g. host="*" -->
+    <httpEndpoint id="defaultHttpEndpoint" httpPort="9080" httpsPort="9443"/>
+
+    <library id="derby">
+      <file name="${server.config.dir}/derby/derby.jar"/>
+    </library>
+
+    <dataSource id="DefaultDataSource">
+      <jdbcDriver libraryRef="derby"/>
+      <!-- Example properties referencing an in-memory Derby Embedded database -->
+      <properties.derby.embedded databaseName="memory:defaultdb" createDatabase="create"/>
+    </dataSource>
+
+    <authData id="myAuth" user="dbuser" password="dbpass"/>
+
+    <!-- Default SSL configuration enables trust for default certificates from the Java runtime --> 
+    <ssl id="defaultSSLConfig" trustDefaultCerts="true"/>
+
+    <remoteFileAccess>
+       <writeDir>${server.config.dir}</writeDir>
+    </remoteFileAccess>
+
+    <basicRegistry id="basic">
+       <user name="admin" password="adminpwd"/>
+    </basicRegistry>
+
+    <!-- Assign 'admin' to Administrator -->
+    <administrator-role>
+        <user>admin</user>
+    </administrator-role>
+
+</server>
+----
+
+For this `server.xml` example, you will need to add the Derby JAR to the server configuration or use your own database configuration.
+
+
+1. Configure a Liberty server using the example `server.xml` file for guidance and then start the server. After the server is started, you can check the logs to find the URL to navigate to the Admin Center. In the previous example, you can navigate to the Admin Center using the `https://localhost:9443/adminCenter/` URL.
+
+
+2. In the Admin Center UI, select the **Server Config** tool.
+
++
+[.img_border_light]
+image::img/blog/ac_db_test_server_config.png[Server Config Tool,width=20%,align="center"]
+
+3. Select **server.xml** to edit.
+
++
+[.img_border_light]
+image::img/blog/ac_db_test_serverxml.png[server.xml,width=50%,align="center"]
+
+4. In the **Design > Server** menu, navigate to the resource you want to test and click the **Test** button.
+
++
+[.img_border_light]
+image::img/blog/ac_db_test_resource.png[Select resource,width=50%,align="center"]
+
+5. Choose the type of authentication your application uses:
+
++
+* For applications that use container authentication, choose the **Application authentication** tab and select whether to use default authentication, specify an authentication alias, or choose a login module configuration.
```

Comment:
For applications that use container authentication, choose the **Container authentication**

Reply:
```suggestion
* For applications that use container authentication, choose the **Container authentication** tab and select whether to use default authentication, specify an authentication alias, or choose a login module configuration.
```